docs/README.md

```
@@ -0,0 +1,76 @@
+
+This document provides instructions on how to set up and start a running instance of the [talawa-admin documentation website](https://docs-admin.talawa.io/) on your local system. The instructions are written to be followed in sequence so make sure to go through each of them step by step without skipping any sections.
+
+## Table of Contents
+
+<!-- TOC -->
+
+- [Talawa Admin Documentation Website](#talawa-admin-documentation-website)
+  - [Installation](#installation)
+  - [Table of Contents](#table-of-contents)
+  - [Prerequisites for Developers](#prerequisites-for-developers)
+    - [Install the Required NPM Package Manager](#install-the-required-npm-package-manager)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)